src/ImageSharp/Formats/Webp/Lossy/Vp8EncIterator.cs

```diff
@@ -81,6 +81,8 @@ public Vp8EncIterator(byte[] yTop, byte[] uvTop, uint[] nz, Vp8MacroBlockInfo[]
             this.I4Boundary = new byte[37];
             this.BitCount = new long[4, 3];
             this.Scratch = new byte[WebpConstants.Bps * 16];
+            this.Scratch2 = new short[17 * 16];
+            this.Scratch3 = new int[16];
```

Comment:
I'm kind of fan a fixed sized buffers -- so would try to use these here too.
Yeah they make the object bigger (by the size of the buffer), but it avoids further objects that the GC needs to track, so most of the time it's advantageous.

**Note**: it's unsafe, so there are no bound checks emitted!